README.md

```diff
@@ -53,8 +64,11 @@ If you need to configure `svgo`, you can also create a config file `svgo.config.
 ### Usage
 
 Import as a Solid.js component:
-```
-import MyIcon from './svgs/my-icon.svg';
+
+```tsx
+import MyIcon from './my-icon.svg';
+// or './my-icon.svg' if `defaultAsComponent` is `false`
```

Comment:
```suggestion
// or './my-icon.svg?component' if `defaultAsComponent` is `false`
```

Reply:
I changed `component` to `component-solid`. In my personal case, I need in one project load svg as Astro components in one place, and as Solid components and another. So Vite need to distinguish which plugin to use.